app/helpers/language_helper.rb

```diff
@@ -1,14 +1,16 @@
+RTL_LOCALES = %w[ar he fa].freeze
```

Comment:
Could you put `fa` before `he` so the list is alphabetical?

Reply:
Ha, thanks! Amended my original commit and force pushed